src/NzbDrone.Core/Download/DownloadService.cs

```diff
@@ -109,7 +112,26 @@ public void DownloadReport(RemoteEpisode remoteEpisode)
                 episodeGrabbedEvent.DownloadId = downloadClientId;
             }
 
-            _logger.ProgressInfo("Report sent to {0}. {1}", downloadClient.Definition.Name, downloadTitle);
+            var indexerName = "Unknown";
```

Comment:
This should already be available as `remoteEpisode.Release.Indexer`, is there a reason that can't be used?

Reply:
No, I didn't realize that was there, I was taking how the name was retrieved by the Id in other places I overlooked the release properties. I change it to pull from there.